websites/G/Goodreads/presence.ts

```diff
@@ -18,24 +18,40 @@ presence.on("UpdateData", async () => {
 	else if (document.location.pathname === "/book")
 		presenceData.details = "Browsing books";
 	else if (document.location.pathname.includes("/book/show/")) {
-		// beta layout conditional
 		presenceData.details = "Viewing a book:";
-		if (document.querySelector("#bookTitle") === null) {
-			book = document.querySelector("h1").textContent;
-			author = document.querySelector("span.ContributorLink__name").textContent;
-			presenceData.state = `${book} | by: ${author}`;
-		} else {
-			book = document.querySelector("#bookTitle").textContent;
-			author = document.querySelector(".authorName").textContent;
-			presenceData.state = `${book} | by: ${author}`;
-		}
+
+		book = document.querySelector("h1").textContent;
+		author = document.querySelector("span.ContributorLink__name").textContent;
+		presenceData.largeImageKey = document
+			.querySelector(".BookCover img.ResponsiveImage")
+			.getAttribute("src");
+		presenceData.state = `${book}`;
```

Comment:
Unneeded string template. `book` is already a string.
```suggestion
		presenceData.state = book;
```